source/release-notes/1.4.txt

```diff
@@ -12,7 +12,23 @@ Release Notes for ``mongosync`` 1.4
    :depth: 2
    :class: singlecol
 
-**Version 1.4 Released June 21, 2023**
+Patch Releases
+--------------
+
+.. _1.4.1-c2c-release-notes:
+
+1.4.1 - June 23, 2023
+~~~~~~~~~~~~~~~~~~~~~
+
+Issues fixed:
+
+- Fixes a build misconfiguration so that telemetry information is sent
+  correctly.
+
+.. _1.4.0-c2c-release-notes:
+
+**Version 1.4.0 released June 21, 2023**
```

Comment:
[question] Does it make sense to have the release versions and dates have the same format? Also, I'm finding it a little difficult to distinguish between the releases visually. Maybe a different heading type and/or some additional space would help.